pkg/client/fake/client.go

```diff
@@ -1177,7 +1177,7 @@ func allowsUnconditionalUpdate(gvk schema.GroupVersionKind) bool {
 		case "PodSecurityPolicy":
 			return true
 		}
-	case "rbac":
+	case "rbac.authorization.k8s.io":
```

Comment:
Add both?

Reply:
Why both?

Reply:
@vincepri you mean for backward compatibility?

Reply:
Yes

Reply:
@vincepri updated, thanks!